core/tests/revert-test/tests/revert-and-restart-en.test.ts

```diff
@@ -0,0 +1,253 @@
+import * as utils from 'zk/build/utils';
```

Comment:
This file looks pretty much identical to the `revert-and-restart.test.ts` (at least I tried to find the difference by eye, but failed). Do we really need to copy-pase it? Would it be worse to use an already present `revert-and-restart.test.ts` (and for whatever difference there change the behavior dynamically)?

Reply:
also, it fails now